netlify/landings/studio.yaml

```
@@ -0,0 +1,101 @@
+path: /studio
+title: Stoplight Studio
+color: blue
+relatedTags:
+  - blog-design
+hero:
+  image:
+    alt: Stoplight Studio
+    src: /images/design_header.png
+  subtitle: >-
+    Build first-class APIs with a collaborative, powerful, visual editor that
+    allows you to prototype and share your API within minutes
+  title: 'Stoplight Studio'
+  ctas:
+    - color: green
+      href: ''
+      name: Download Now
+    - color: green
+      href: ''
+      name: Sign Up Now      
+featureSection:
+  actionBar:
+    buttons:
+      - href: 'https://next.stoplight.io/'
+        title: Get Started
+    enabled: true
+    text: Ready to Model and Design?
+  title: OpenAPI v2/v3 Visual Editor
+  description: >-
+    Start designing your API in minutes. Use the [OpenAPI
+    Specification](/api-design-guide/oas-spec/) (formerly known as Swagger) with
+    Stoplight Studio to design consistent and standardized HTTP APIs for your 
+    organization. Without writing any code, model complex APIs faster than ever 
+    while simultaneously getting feedback on your prototypes with our 
+    [instant mock servers](/mocking/).
+  features:
+    - description: >-
+        Design an API with speed and efficiency from scratch. Stoplight Studio 
+        decreases the learning curve so you don’t need to be an OpenAPI
+        expert to create first-class API designs. Describe endpoints, headers,
+        bodies, multiple responses, query string parameters, shared models, and 
+        examples, and much more for complex APIs.
+      image: 
+      shortName: OpenAPI Visual Editor
+      title: 'Powerful, OpenAPI v2/v3 Visual Editor'
+
+    - description: >-
+        Instantly prototype and collaborate on your API design with our integrated, 
+        instant Mock Servers, powered by [Prism](/prism). Mock a single API, multiple APIs, 
+        or even just a few endpoints. Increase efficiency by allowing your frontend teams to start
+        implementation while the backend team develops the API.
+      image: 
+      shortName: Mock Servers
+      title: 'Instant Mock Servers'
+
+    - description: >-
+        Create API documentation in minutes. Combine OpenAPI documents
+        (formerly known as Swagger) with Markdown to create robust, searchable,
```

```suggestion
        with Markdown to create robust, searchable,
```

We mention this up on https://github.com/stoplightio/stoplight.io/pull/1046/files#diff-23cbee1bf85fc882b142c82a34af32d7R31 

Maybe once is enough?